docs/docs/building-apps/04-vote-extensions.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 1
+---
+
+# Vote Extensions
+
+:::note Synopsis
+This sections describes how the application can define and use vote extensions
+defined in ABCI++.
+:::
+
+## Extend Vote
+
+ABCI++ allows an application to extend a pre-commit vote with arbitrary data. This
+process does NOT have be deterministic and the data returned can be unique to the
+validator process. The Cosmos SDK defines `ExtendVoteHandler`:
+
+```go
```

Comment:
```md
```go reference
https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.0/types/abci.go#L26-L27
```
```

Reply:
Can you make a direct code suggestion pls?